foundry/socketInterface.d.ts

```diff
@@ -47,7 +39,6 @@ declare namespace SocketInterface {
     }
   }
 
-  // TODO: go through all SocketInterface.dispatch calls and collect requests
```

Comment:
If you leave in the other TODO, you might as well leave this as well.

Reply:
that was unintentional, I'll readd it

Reply:
Added it again